src/openforms/conf/base.py

```diff
@@ -1121,6 +1121,10 @@
     + config("CSP_EXTRA_IMG_SRC", default=[], split=True)
 )
 
+# affects <object> and <embed> tags, block everything by default but allow deploy-time
+# overrides.
+CSP_OBJECT_SRC = config("CSP_OBJECT_SRC", default=["'none'"], split=True)
```

Comment:
Funny... passing `split=False` would do the same.

Reply:
Created an internal taiga issue for this one :grimacing: